components/resources/brave_news_strings.grdp

```diff
@@ -178,4 +178,45 @@
   <message name="IDS_BRAVE_NEWS_CHANNEL_WORLD_NEWS" desc="Translation of the 'World News' channel">
     World News
   </message>
+
+  <!-- Feed -->
+  <message name="IDS_BRAVE_NEWS_NEWS_PEEK" desc="The 'News' heading for the peeking card">
+    News
+  </message>
+  <message name="IDS_BRAVE_NEWS_MY_FEED_HEADING" desc="The heading for the feed sidebar">
+    My Feed
+  </message>
+  <message name="IDS_BRAVE_NEWS_FOR_YOU_FEED" desc="The heading for For You feed">
+    For You
+  </message>
+  <message name="IDS_BRAVE_NEWS_FOLLOWING_FEED" desc="The heading for Following feed">
+    Following
+  </message>
+  <message name="IDS_BRAVE_NEWS_ADD_CHANNELS_OR_PUBLISHERS" desc="Text for the buttons for adding channels or adding publishers">
+    Add
+  </message>
+  <message name="IDS_BRAVE_NEWS_PUBLISHERS_HEADING" desc="Heading for the list of followed publishers">
+    Publishers
+  </message>
+  <message name="IDS_BRAVE_NEWS_SHOW_ALL" desc="Text for the button to show all publishers or channels">
+    Show all
+  </message>
+  <message name="IDS_BRAVE_NEWS_SHOW_LESS" desc="Text for the button to show fewer publishers or channels">
+    Show less
+  </message>
+  <message name="IDS_BRAVE_NEWS_ADVERT_BADGE" desc="Text for the badge indicating advertisements. Should be short">
+    Ad
```

Comment:
Not sure we can guarantee the shortness of localizations.

Reply:
The design should deal with it even if it's a bit longer :smile: I'm wondering if this should even be translated now?